core/src/main/java/io/seata/core/constants/ConfigurationKeys.java

```diff
@@ -154,22 +162,22 @@
     /**
      * The constant STORE_DB_GLOBAL_TABLE.
      */
-    public static final String STORE_DB_GLOBAL_TABLE = STORE_DB_PREFIX + "global.table";
+    public static final String STORE_DB_GLOBAL_TABLE = STORE_DB_PREFIX + "globalTable";
 
     /**
      * The constant STORE_DB_BRANCH_TABLE.
      */
-    public static final String STORE_DB_BRANCH_TABLE = STORE_DB_PREFIX + "branch.table";
+    public static final String STORE_DB_BRANCH_TABLE = STORE_DB_PREFIX + "branchTable";
 
     /**
      * The constant STORE_DB_GLOBAL_DEFAULT_TABLE.
      */
-    public static final String STORE_DB_GLOBAL_DEFAULT_TABLE = "global_table";
+    public static final String STORE_DB_GLOBAL_DEFAULT_TABLE = "globalTable";
 
     /**
      * The constant STORE_DB_BRANCH_DEFAULT_TABLE.
      */
-    public static final String STORE_DB_BRANCH_DEFAULT_TABLE = "branch_table";
+    public static final String STORE_DB_BRANCH_DEFAULT_TABLE = "branchTable";
```

Comment:
globalTable->global_table, branchTable->branch_table
and move them to DefaultValues.java